### PR TITLE
We are databasing!

### PR DESCRIPTION
**Pull request by justinsprecco**
# Initial Database

## Database

Created a starter database for the metadata, users, and chat.

## Packages

Added the mysql package for the database, and dotenv for local setups.
The package-lock lists Denque is a dependency for mysql2.

## README

README explains the details on how to setup this database.

## .gitignore

Added the new .env to the gitignore to hide our secret identities

### CODE REVIEW

**Review by joebarteluce — APPROVED** [review]

File: db/schema.sql [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+CREATE TABLE Users (
+    id INT AUTO_INCREMENT PRIMARY KEY,
+    username VARCHAR(255) NOT NULL UNIQUE,
+    email VARCHAR(255) NOT NULL UNIQUE,
+    password_hash VARCHAR(255) NOT NULL,
+    registration_date TIMESTAMP DEFAULT CURRENT_TIMESTAMP
+);
+
+-- Create Streams table
+CREATE TABLE Streams (
```
[the remark this answers is not in the document]
Comment (Da-Scher) [reply db/schema.sql:16]: Note, change it to `photo VARCHAR(255)`

**Review by Da-Scher — CHANGES_REQUESTED** [review]
it looks good, but Joe needs that photo support for that smooth gallery :)

File: db/schema.sql (hunk 1 above)
[the remark this answers is not in the document]
Comment (Da-Scher) [reply db/schema.sql:16]: Fix made; thank you for your patience

**Review by Da-Scher — APPROVED** [review]
Requested changes made.
LGTM :+1:

**Pull request merged**